step.yml

```diff
@@ -181,14 +181,25 @@ inputs:
         - "up_until_maximum_repetitions"
   - maximum_test_repetitions: 3
     opts:
-      title: Maximum Test Repetitions
+      title: Maximum Test Repetitions (Available in Xcode 13+)
       category: Test Repetition
       summary: The maximum number of times a test will repeat based on Test Repetition Mode.
       description: |-
         The maximum number of times a test will repeat based on Test Repetition Mode.
         
         Should be more than 1 if Test Repetition Mode (`test_repetition_mode`) is other than `none`.
       is_required: true
+  - relaunch_tests_for_each_repetition: "no"
+    opts:
+      title: Relaunch Tests for Each Repetition (Available in Xcode 13+)
```

Comment:
same as above but up to you

Reply:
Let's continue the discussion in that thread about this topic!